test.sh

```diff
@@ -71,8 +71,8 @@ function delete_release() {
   echo "=========================================="
   echo "Deleting Conjur Helm release $RELEASE_NAME"
   echo "=========================================="
-  if [ ! -z "HELM_DEL_ARGS" ]; then 
-    helm del "$HELM_DEL_ARGS " "$RELEASE_NAME"
+  if [ ! -z "$HELM_DEL_ARGS" ]; then
+    helm del $HELM_DEL_ARGS "$RELEASE_NAME"
```

Comment:
do you want to drop the quotes around the `HELM_DEL_ARGS` var?

Reply:
Yes, I need to expand $HELM_DEL_ARGS as multiple arguments, not as a single string argument.